src/Sulu/Bundle/MediaBundle/Media/Video/FFMPEGToolBox.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/*
+ * This file is part of the Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Media\Video;
+
+class FFMPEGToolBox implements FFMPEGToolBoxInterface
```

Comment:
I would call this class `VideoUtils` or something like this... And in general, isn't there a ready to use library or bundle to use, instead of implementing stuff like this on your own?